spec.bs

```diff
@@ -4884,15 +4885,21 @@ JavaScript is controlled and limited as follows:
 - Responses are required to contain the `Ad-Auction-Allowed: ?1` header.
 - Fetches are uncredentialed.
 
-Protected Audience has the browser pass in several “browserSignals” to the bidding script that give the script
-unforgeable information about the context that the script is being executed in. This way bidders
-and sellers have the choice to only participate in auctions where they are comfortable working with
-the involved parties.
+Protected Audience has the browser pass in several “browserSignals” to the bidding script that give
+the script unforgeable information about the context that the script is being executed in. This way
+bidders and sellers have the choice to only participate in auctions where they are comfortable
+working with the involved parties.
 
-The [execution environment](#script-runners) available to these scripts is the absolute minimum necessary to calculate
-
-a bid. It supports only ECMAScript. It does not support network, storage, timer, date, DOM,
-Workers, postMessage, Navigator or Window APIs.
+The [execution environment](#script-runners) available to these scripts is the absolute minimum
+necessary to calculate a bid. It supports only ECMAScript. It does not support network, storage,
+timer, date, DOM, Workers, postMessage, Navigator or Window APIs.
 
 Protected Audience adds Permission-Policies to control access to the Protected Audience APIs to
 give sites and embedders the ability to clamp down on use of the APIs as they see fit.
+
+Auction config and interest group data are processed as JSON and not via the [=structured clone
+algorithm=], in part because interest groups are updated via JSON server responses. However, there
+is also a security reason to not use the structured clone algorithm, which is that the data may
+be shared with other parties -- in particular, the auction config perBuyerSignals are sent from the

```

Comment:
Can you link to `perBuyerSignals` directly here?

Reply:
Done